tests/test_parameters.py

```diff
@@ -205,6 +205,7 @@ def test_dumps_loads_parameters(self):
         pars.add('x', value=1.0)
         pars.add('y', value=2.0)
         pars['x'].expr = 'y / 2.0'
+        pars['x'].expr = 'y / 2.0'
```

Comment:
this seems redundant as it's the same as the line above?

Reply:
@reneeotten  Thanks, I think I addressed all these...